pyproject.toml

```diff
@@ -1,9 +1,67 @@
 [build-system]
-requires = ["setuptools>=42"]
+requires = ["setuptools>=42", "pytest-runner", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "dpsim"
+version = "1.1.1"
+description = "Dynamic real-time power system simulator"
+readme =  {file = "README.md", content-type = "text/markdown"}
+authors = [
+    {name = "The DPsim Authors", email = "mmirz@eonerc.rwth-aachen.de"},
```

Comment:
@m-mirz Is this email address still valid?

Reply:
@martinmoraga no it is not. @stv0g It should be markus.mirz@rwth-aachen.de

Reply:
I will fix it.